packages/examples/packages/bip32/src/index.ts

```diff
@@ -82,6 +86,14 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
         return bytesToHex(signed);
       }
 
+      if (curve === 'ed25519bip32') {
```

Comment:
Could we just use the logic above instead? 
`if (curve === 'ed25519 || curve === 'ed25519bip32')`

Reply:
that throws an error for unexpected constant condition

Reply:
this change has been pushed